whac-a-mole.py

```diff
@@ -44,6 +49,8 @@ def onwhack():
     global whacks
     whacks += 1
     scorelabel.config(text="* " + str(whacks) + " *")
+    playsound('sounds/ow.wav')

```

Comment:
Note that when this sound plays, the mole should turn red, with the mole's text becoming `*_*` simultaneously. I think you might have to use a basic [threading](https://docs.python.org/3/library/threading.html) here to make the two functions run simultaneously.

Reply:
Will add this